src/scss/foundations/breakpoints.scss

```diff
@@ -0,0 +1,65 @@
+@use "../lib";
+
+$_breakpointBase: 1px;
+$_breakpointXsmall: 576px;
+$_breakpointSmall: 768px;
+$_breakpointMedium: 992px;
+$_breakpointLarge: 1200px;
+$_breakpointXlarge: 1920px;
+
+/**
+ * The breakpoints of Pyxis
+ *
+ * @type Map
+ * @access private
+ */
+$_breakpoints: (
+  base: $_breakpointBase,
+  xsmall: $_breakpointXsmall,
+  small: $_breakpointSmall,
+  medium: $_breakpointMedium,
+  large: $_breakpointLarge,
+  xlarge: $_breakpointXlarge
+);
+
+/**
+ * Breakpoint bounding difference
+ *
+ * @type Value
+ * @access private
+ */
+$_breakpointBoundDiff: 1px;
+
+/**
+ * Returns the $breakpoint value from map $breakpoints
+ *
+ * @param {String} $breakpoint - Name of breakpoint
+ * @return {Value} Representation of breakpoint color
+ * @throws {Error}
+ *
+ * @access public
+ */
+@function breakpoint($breakpoint) {
+  @return lib.mapGetOrDie($_breakpoints, $breakpoint);
```

Comment:
Possiamo stringare il nome `mapGetOrDie` in `getOrDie`?

Reply:
Per ora direi di si, poi se ci accorgiamo che è troppo generico e ce ne serve uno per le mappe e uno per le stringhe (dico a caso) li facciamo più specifici